vars.mk

```diff
@@ -1,5 +1,5 @@
 PKG_NAME := github.com/docker/app
-BIN_NAME := docker-app
+BIN_NAME ?= docker-app
```

Comment:
I believe this is non needed anymore.

Reply:
Well it depends if the CI is friendly enough to let it build without it 😇

Reply:
You're still hijacking `BIN_NAME` to build `yamlschema` from the `Jenkinsfile`?

Reply:
What else? ☕️

Reply:
Still there it seems? 🐨